datafeeder/src/main/java/org/georchestra/datafeeder/api/DataPublishingApiController.java

```diff
@@ -53,6 +55,8 @@ public class DataPublishingApiController implements DataPublishingApi {
     private @Autowired DataUploadService uploadService;
     private @Autowired DataPublishingResponseMapper mapper;
     private @Autowired DataFeederConfigurationProperties props;
+    private @Autowired GeorchestraNameNormalizer nameResolver;
+    private @Autowired(required = false) PostgisSchemasConfiguration postgisSchemasConfiguration;
```

Comment:
there are some guidelines which indicate that we should not use Autowired, but since it looks like it is already the case here, I won't object (At least I'm pretty sure we had a talk about that with @groldan).